### PR TITLE
Edge x 3.0 update av cleanup env

### PR DESCRIPTION
**Pull request by seanohair22**
## PR Checklist

- [ ] Added label to the Pull Request for easier discoverability and search
- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-vending/blob/main/.github/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented out code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Cleans up AV Compose file. 
Cleans up Makefile clean target. 
Cleans up ms-authentication to use automated-vending instead of automated-checkout.

## Issue this PR will close

closes: #127
close: #**127**

## Test Instructions if applicable

 - cd automated-vending
 - Run `make docker`
 - Run `make run`
 - Check containers are up and running via Portainer.
 - Run `make clean`

### CODE REVIEW

File: docker-compose.av.yml [hunk 1]
```diff
@@ -145,12 +134,13 @@ services:
...
     image: "automated-vending/ds-cv-inference:dev"
-    container_name: automated-checkout_ds-cv-inference
+    container_name: automated-vending_ds-cv-inference
     command: ["/go/src/ds-cv-inference/images","mqtt-broker:1883","0.85","/go/src/ds-cv-inference/skumapping.json"]
     depends_on:
       mqtt-broker:
         condition: service_started
     environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
       SERVICE_HOST: ds-cv-inference
```
Comment (lenny-intel) [inline docker-compose.av.yml:144]: This is not an EdgeX based service so these emv can also be removed.

File: docker-compose.av.yml [hunk 2]
```diff
@@ -175,7 +165,6 @@ services:
       consul:
         condition: service_started
     environment:
-      CLIENTS_CORE_COMMAND_HOST: edgex-core-command
```
Comment (lenny-intel) [inline docker-compose.av.yml:178]: Service does use the command client, so this is needed.

File: docker-compose.av.yml [hunk 3]
```diff
@@ -206,13 +195,8 @@ services:
...
       CLIENTS_CORE_COMMAND_HOST: edgex-core-command
-      CLIENTS_CORE_DATA_HOST: edgex-core-data
-      CLIENTS_CORE_METADATA_HOST: edgex-core-metadata
       CLIENTS_SUPPORT_NOTIFICATIONS_HOST: edgex-support-notifications
       EDGEX_SECURITY_SECRET_STORE: "false"
-      TRIGGER_EDGEXMESSAGEBUS_PUBLISHHOST_HOST: mqtt-broker
-      TRIGGER_EDGEXMESSAGEBUS_SUBSCRIBEHOST_HOST: mqtt-broker
-      REGISTRY_HOST: edgex-core-consul
       SERVICE_HOST: as-controller-board-status
       CONTROLLERBOARDSTATUS_DOORSTATUSCOMMANDENDPOINT: "http://edgex-core-command:59882/api/v2/device/name/Inference-device/inferenceDoorStatus"
```
Comment (lenny-intel) [inline docker-compose.av.yml:201]: The setting has been replace by use of the command client.
```suggestion
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: docker-compose.av.yml (hunk 1 above)
Comment (seanohair22) [reply docker-compose.av.yml:144]: Just the EDGEX_SECURITY_SECRET_STORE or the SERVICE_HOST also?

Comment (lenny-intel) [reply docker-compose.av.yml:144]: Both

**Review by jim-wang-intel — APPROVED** [review]
LGTM except Lenny's comments

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**